gui/wxpython/modules/import_export.py

```diff
@@ -270,6 +274,30 @@ def _validateOutputMapName(self):
                 return False
         return True
 
+    def _handleCloseEvent(self, *args, **kwargs):
+        if len(self._commands_running) > 0:
+            # prevent dialog close while async commands are running
+            return
```

Comment:
This should probably use event.Veto:
https://docs.wxpython.org/wx.CloseEvent.html